igvm/vm.py

```diff
@@ -197,6 +201,47 @@ def start(self, force_stop_failed=True, transaction=None):
         if transaction:
             transaction.on_rollback('stop VM', self.shutdown)
 
+    def aws_start(self):
+        """AWS start
+
+        Start a VM in AWS.
+        """
+
+        ec2 = boto3.client('ec2')
+
+        try:
+            ec2.start_instances(
+                InstanceIds=[self.dataset_obj['aws_instance_id']],
+                DryRun=True
+            )
+        except ClientError as e:
+            if 'DryRunOperation' not in str(e):
```

Comment:
Can this information really be only obtained by parsing a string?

Reply:
Yes, it can:

> Parameters
> DryRun (boolean) -- Checks whether you have the required permissions for the action, without actually making the request, and provides an error response. If you have the required permissions, the error response is DryRunOperation . Otherwise, it is UnauthorizedOperation .

Reference: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html

Reply:
We are hiding the DryRunOperation, but will raise UnauthorizedOperation e.g.